skhep_testdata/__main__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import print_function
```

Comment:
Could you add somewhere how one should use this `__main__` module? I see no usage anyway, unless I overlooked it. It could be made a script in the same way the `scikit-hep` package has a trivial example of a skhep-XX script now.

Reply:
You could also add some trivial doc to the file.

Reply:
When you do:
```
python -m <module>
```
it looks for `__main__.py` and runs that.  So adding this file adds the new functionality  that I describe in the README:
```bash
 python -m skhep_testdata cms_hep_2012_tutorial/data.root 
```

Reply:
As for making it an explicit executable.  I can definitely do that, but I was thinking this way avoids polluting the user with extra commands.  With this file included, it's very easy to set that up as well from the `setup.py` using the `entry_points` option.

Reply:
Good point. So just giving the example usage in the doc at the top of `__main__.py` could be useful for those browsing the repo. Just a suggestion.